examples/online_serving/whisper_api_client.py

```diff
@@ -0,0 +1,88 @@
+# SPDX-License-Identifier: Apache-2.0
+import argparse
+import asyncio
+import json
+from subprocess import CalledProcessError, run
+
+import aiohttp
+import numpy as np
+
+AIOHTTP_TIMEOUT = aiohttp.ClientTimeout(total=6 * 60 * 60)
+SAMPLE_RATE = 16000
+
+
+def load_audio_from_file(file: str, sample_rate: int = SAMPLE_RATE):
+    cmd = [
+        "ffmpeg", "-nostdin", "-threads", "0", "-i", file, "-f", "s16le",
+        "-ac", "1", "-acodec", "pcm_s16le", "-ar",
+        str(sample_rate), "-"
```

Comment:
Might be worth it to output this to a temporary file in `/tmp/`, load it from there and then delete the temporary file rather than streaming this to stdout.

Reply:
true